test/bdd/bddtests_test.go

```diff
@@ -196,7 +195,6 @@ func features() []feature {
 		outofband.NewOutofbandControllerSteps(),
 		presentproof.NewPresentProofSDKSteps(),
 		presentproof.NewPresentProofControllerSteps(),
-		redeemableroutes.NewBDDSteps(),
```

Comment:
what happened to this?

Reply:
The `introduce` BDD test [here](https://github.com/hyperledger/aries-framework-go/pull/2729#discussion_r613715014) is intended to demonstrate the same thing.